service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/ServiceClientOptions.java

```diff
@@ -14,4 +16,12 @@
      */
     @Getter
     private ProxyOptions proxyOptions;
+
+
+    /**
+     * The SSL context to use when opening the AMQPS/AMQPS_WS connections. If not set, this library will generate the default
+     * SSL context that trusts the IoT Hub public certificates.
```

Comment:
ah, I see that you've mentioned this here :)
maybe we should add this to the public API javadoc as well